content/advanced/430_emr_on_eks/eks_spark_new_york_taxidataset.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "Example - Running Spark jobs on NY Taxi Data"
+date: 2021-05-19T10:30:14-04:00
+weight: 61
+draft: true
+---
+
+In this chater, you will use spark jobs running on EKS to analyze the New York City Taxi and Limousine Commision (TLC) Trip Record Data . The data is available at Registry of Open Data on AWS at https://registry.opendata.aws/nyc-tlc-trip-records-pds/ . For calendar years 2019 and 2020, you will calculate top-10 pickup locations in New York area based on total amount of distance travelled by the taxi's.  
+
+
+You can find more details about Yellow Cabs and Green Cabs at https://www1.nyc.gov/site/tlc/businesses/yellow-cab.page and https://www1.nyc.gov/site/tlc/businesses/green-cab.page.
+
+
+### Creating Nodegroup
```

Comment:
I think context behind using node selectors is missing. I'd write paragraph on how node selectors help and which use cases benefit from node selectors (for ex, single-az, running executors on spot while drivers use on-demand etc)